core/templates/pages/contributor-dashboard-page/modal-templates/translation-modal.component.ts

```diff
@@ -280,11 +280,14 @@ export class TranslationModalComponent {
   hasSomeDuplicateElements(
       originalElements: string[],
       translatedElements: string[]): boolean {
+    const mathEquationRegex = new RegExp(
+      /(?:(?:^|[-+_*/=])(?:\s*-?\d+(\.\d+)?(?:[eE][+-]?\d+)?\s*))+$/);
```

Comment:
This regex seems pretty complicated, can you add a comment explaining what this should cover, e.g. digits, operators, exponents, etc.

Reply:
Done

Reply:
What is meant by "(including equal)/number pairs"? I think it'd be good to include a few examples as well.

Reply:
Done and updated the comment. **including equal** means numeric captions that includes equals symbol are also filtered by this regex.